e2e/features/homeoffice-upload-evidence.feature

```diff
@@ -0,0 +1,64 @@
+Feature: Admin officer has reduced functionality
```

Comment:
I'd look to amend this description for the Home Office user

Reply:
Thanks. changed.